src/py/mat3ra/made/tools/calculate.py

```diff
@@ -125,3 +130,40 @@ def calculate_interfacial_energy(
     surface_energy_film = calculate_surface_energy(film_slab, film_bulk, calculator)
     adhesion_energy = calculate_adhesion_energy(interface, substrate_slab, film_slab, calculator)
     return surface_energy_film + surface_energy_substrate - adhesion_energy
+
+
+@decorator_handle_periodic_boundary_conditions(cutoff=0.25)
+def calculate_norm_of_distances(material: Material, shadowing_radius: float = 2.5) -> float:
```

Comment:
calculate_film_substrate_interaction_metric, should have `interaction_function` as parameter with `um_of_inverse_distances_squared` as default